README.md

```diff
@@ -718,7 +718,16 @@ default version of the stub is Python 3.8.
 provide the path to the tar file in the `EXECUTION_ENV_PATH` in the
 `config.pbtxt` of all the models that want to use the execution environment.
 
-4. If you need to compile the Python backend stub, it is recommended that you
+4. The `EXECUTION_ENV_PATH` **should not** point to anywhere outside the
+`$$TRITON_MODEL_DIRECTORY`, the behavior of accessing anywhere outside the
+`$$TRITON_MODEL_DIRECTORY` is **undefined**.
+
+5. An *experimental* feature has been added to allow the `EXECUTION_ENV_PATH`
+to access one level outside the `$$TRITON_MODEL_DIRECTORY` for local and S3
+storages. The *experimental* feature can be modified or removed without any
```

Comment:
"The experimental.." -> "This experimental..."

Reply:
Add this sentence in the end. "Note that this experimental feature does NOT work for Azure Cloud Storage (ACS) or Google Cloud Storage (GCS)."

Reply:
Just not just don't mention this at all?

Reply:
Yes. I think not mentioning this at all can simplify things, because undefined behavior can cover it all.
This sentence is removed.
@Tabrizian please comment if you think otherwise.

Reply:
Sounds good to me too.